Doc/library/unittest.mock-examples.rst

```diff
@@ -360,6 +360,30 @@ of arbitrary attributes as well as the getting of them then you can use
 *spec_set* instead of *spec*.
 
 
+Using side_effect to return per file content
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+:func:`mock_open` is used to patch :func:`open` method. :attr:`~Mock.side_effect`
+can be used to return a new Mock object per call. This can be used to return different
+content per file stored in a dictionary::
+
+    DEFAULT = "default"
```

Comment:
This should be initially indented 3 spaces as convention (see the example `def f(a, b, c): pass`) above.